Lokad.AzureEventStore.Cache/Lokad.AzureEventStore.Cache.csproj

```diff
@@ -14,19 +14,19 @@
 
   <!-- When working locally, comment out this block and uncomment the one below. -->
 
-  <ItemGroup>
+  <!--<ItemGroup>
```

Comment:
Comments  need to be restored.

Reply:
Done!, You may want to bump up the `<PackageReference>` versions after publishing this new version to NuGet.